app/mailers/response_mailer.rb

```diff
@@ -2,8 +2,21 @@ class ResponseMailer < ApplicationMailer
   def response_email(invitation_sent_at:, requester_email:, show_secret:)
     @invitation_sent_at = invitation_sent_at
     @link = show_response_url(showSecret: show_secret)
+    @show_secret = show_secret
+
+    attachments.inline['Rapport-CATja-Screening.pdf'] = render_report
 
     mail to: requester_email,
          subject: 'Resultaten invulling'
   end
+
+  private
+
+  def response_by_show_secret
```

Comment:
Misschien is het in de nieuwe situatie die deze PR creërt simpeler om als argument aan `response_email` de hele response mee te geven i.p.v. alleen de `show_secret`. Dan hoeft er geen logic om de response op te zoeken in ResponseMailer.

Slechts een idee, take it or leave it :)